.github/dependabot.yml

```diff
@@ -3,32 +3,26 @@
 version: 2
 updates:
   # Check for updates to GitHub Actions every month
-  - package-ecosystem: 'github-actions'
-    directory: '/'
+  - package-ecosystem: "github-actions"
```

Comment:
https://github.com/grafana/plugin-tools/blob/main/.prettierrc#L7 singleQuote or double quote?

Reply:
It should really be single quotes. I'm hoping to revert this revert soon enough. 😄